public/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+apigility-ui/
+zf-apigility/
```

Comment:
I added it intentionally so that if the user decides to keep git history, it doesn't immediately show up as a new file.